exporter/exporter.collect.go

```diff
@@ -26,15 +26,15 @@ func (e *Exporter) CollectPrometheusMetric(ch chan<- semp.PrometheusMetric) {
 			} else {
 				up = 0
 				err = errors.New("Software only scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
-				_ = level.Error(e.logger).Log("Hardware only  scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
+				_ = level.Error(e.logger).Log("Software only  scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
 			}
 		case "StorageElement", "StorageElementV1":
 			if !e.config.IsHWBroker {
 				up, err = e.semp.GetStorageElementSemp1(ch, dataSource.ItemFilter)
 			} else {
 				up = 0
 				err = errors.New("Software only scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
```

Comment:
This line has to be changed as well.
`
```suggestion
				err = errors.New("Hardware only scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
```

Reply:
StorageElement is a software only target, so I think that is correct?